backend_entrypoint.sh

```diff
@@ -50,7 +80,24 @@ cmd_run() {
         sleep 10
     done
 
-    exec supervisord -c "supervisord/$1.conf"
+    supervisord_includes=$(_get_supervisord_includes "${@:2}")
```

Comment:
It seems to me that, instead of accepting the list of includes as a parameter, this script should determine it automatically based on the list of workers. Otherwise, you're going to duplicate the includes everywhere the worker containers are configured (e.g. in the Compose file and the Helm chart), which could lead to desynchronization over time.